c/src/detectors/c-out-of-bounds-write/c-out-of-bounds-write_compliant.c

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// {fact rule=c-out-of-bounds-write@v1.0 defects=0}
+#include <stdio.h>
+
+int compliant(){
+    //Compliant : Perform bounds checking, validate input sizes and indices to ensure they stay within the allocated memory bounds.
```

Comment:
1. The comment is misplaced
2. Please shorten the comment to a single line if possible